src/rust/jsg/v8.rs

```diff
@@ -24,6 +24,16 @@ pub mod ffi {
         Error,
     }
 
+    /// Module visibility level, mirroring workerd::jsg::ModuleType from modules.capnp.
+    ///
+    /// CXX shared enums cannot reference existing C++ enums, so we define matching values here.
+    /// The conversion to workerd::jsg::ModuleType happens in jsg.h's RustModuleRegistry.
+    enum ModuleType {
```

Comment:
you should reuse existing enum: and not assign values

https://github.com/cloudflare/workerd/blob/main/src/rust/kj/http.rs#L27